k-distribution/src/main/scripts/bin/krun

```diff
@@ -215,7 +215,9 @@ do
       # slash removes prefixes so build a new array with the files to keep
       newArray=()
       for value in "${tempFiles[@]}"; do
-          $value != $tempDir && newArray+=($value)
+          if [ $value != $tempDir ]; then 
+            newArray+=($value)
+          fi
```

Comment:
Just something to note (for posterity), you should never use `&&` to chain together commands (for the reason you noticed). 

Instead `||` should be used. So you wolud instead say:

```
$value == $tempDir || newArray+=($value)
```

This makes sure that the line never throws an error.

But it looks good!

Reply:
@ehildenb I thought that if you use `||`, the command on the right only gets executed when the first one fails. If that's the case, then I'm not sure `||` is appropriate for chaining commands in general, unless I'm missing something...